File: _release/bitmap/array.test.php
<?php
ini_set ('memory_limit', '2048M');

$million = 1000000;
$million10 = 100 * $million;
$billion = 1000 * $million;
$large_number =  4 * $billion;

$pos = 0;
$data = array();
for($i=0;$i<$large_number;$i++){
	if($i % $million == 0){
		echo ".";
	}
	if($i % $million10 == 0){
		echo "\n".date("Y-m-d H:i:s")."\n";
	}
	$data[$i] =1 ;
}

File: _release/bitmap/array.test.cpp
#include <stdio.h>
#include <string.h>
#include <stdlib.h>
#include <stdint.h>


int main(){
	int million = 1000000;//1百万
	int million10 = 10 * million;//千万
	unsigned long billion = 1000 * million;//10亿
	unsigned long large_number = 4 * billion;
	int * int_array[million10];

	printf("%u,%u,%u,%u\n",million,million10,billion,large_number);
}